pages/resources/discovery.mdx

```diff
@@ -0,0 +1,443 @@
+# Discovery
+
+### Searching Discovery
+
+Discord utilizes [Algolia](https://www.algolia.com/) to power search for discovery. You can search for guilds by name, description, keywords, and more. Reference the [Algolia Search documentation](https://www.algolia.com/doc/rest-api/search/) for more information on how to search. A [proxied version of the Algolia Search API](#search-discoverable-guilds) is available, with limited parameter support.
+
+<Alert type="info">
+
+All hits returned from Algolia are [discoverable guild](#discoverable-guild-object) objects. However, some fields such as `features` are truncated to only include values that are relevant to discovery.
```

Comment:
Specify the features that are relevant if known pls

Reply:
I don't think its worth maintaining a list for that.